test/joke_test.rb

```diff
@@ -3,9 +3,19 @@
 require './lib/joke'
 
 class JokeTest < Minitest::Test
+  def setup
+    @joke = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+  end
+
   def test_it_exists
+
+    assert_instance_of Joke, @joke
   end
 
   def test_it_has_attributes
+
+    assert_equal 1, @joke.id
+    assert_equal "Why did the strawberry cross the road?", @joke.setup
+    assert_equal "Because his mother was in a jam.", @joke.punchline
   end
```

Comment:
good job in here and nice use of setup; be sure to clear out those blank lines on line 11 and 16 afterward